models/iaf_psc_alpha.h

```diff
@@ -149,6 +149,12 @@ See also
 
 iaf_psc_delta, iaf_psc_exp, iaf_cond_exp
 
+
+Examples using this model
++++++++++++++++++++++++++
+
+.. listexamples:: iaf_psc_alpha
```

Comment:
Should we also add this for all the other models?

Reply:
In principle yes, the main goal of this PR is to create the extension. I have only added it to 2 models, to see if the logic works and the correct output is displayed, but I can add it to all.

Reply:
Thanks, Jessica!